src/stratigraphy/benchmark/score.py

```diff
@@ -177,125 +95,17 @@ def get_metrics(predictions: dict[str, FilePredictions], field_key: str, field_n
         field_name (str): The name of the field being evaluated.
 
     Returns:
-        dict: The document level metrics and overall metrics.
+        dict: The requested DatasetMetrics object.
```

Comment:
Return type should be DatasetMetrics or am I mistaken?

Reply:
yes, fixed